packages/list-plugins/lib/tracking/byTracking.js

```diff
@@ -0,0 +1,65 @@
+const { Relationship } = require('@keystone-alpha/fields');
+const { composeResolveInput } = require('../utils');
+
+const _byTracking = ({ created = true, updated = true }) => ({
+  updatedByField = 'updatedBy',
+  createdByField = 'createdBy',
+  ...byFieldOptions
+} = {}) => ({ fields = {}, hooks = {}, ...rest }) => {
+  const relationshipOptions = {
+    type: Relationship,
+    ref: 'User',
+    access: {
+      read: true,
+      create: false,
+      update: false,
+    },
+    ...byFieldOptions,
+  };
+
+  if (updated) {
+    fields[updatedByField] = {
+      ...relationshipOptions,
+    };
+  }
+
+  if (created) {
+    fields[createdByField] = {
+      ...relationshipOptions,
+    };
+  }
+
+  const newResolveInput = ({ resolvedData, existingItem, originalInput, context }) => {
+    if (Object.keys(originalInput).length === 0) {
+      return resolvedData;
+    }
+    const { authedItem: { id = null } = {} } = context;
+    if (existingItem === undefined) {
+      // create mode
+      if (created) {
+        resolvedData[createdByField] = id;
+      }
+      if (updated) {
+        resolvedData[updatedByField] = id;
+      }
+    } else {
+      // update mode
+      if (created) {
+        delete resolvedData[createdByField]; // createdByField No longer sent by api/admin, but access control can be skipped!
```

Comment:
@MadeByMike this is no longer sent from admin/graphql as addressed in #1521 
but may still be sent by skipping access control (not tested)

Reply:
It can't hurt to leave it here just incase it is ever sent. But if you're confident you can remove it.

Reply:
will leave it for now unless tested later.